moon/powerups/server/viper.moon

```diff
@@ -44,20 +42,22 @@ class ViperPowerup extends BasePowerup
             attacker = dmg\GetAttacker!
 
             if not IsValid(attacker) and attacker\IsPlayer! return
-            if attacker ~= @owner return
+            return unless attacker == @owner
 
             attackerWeapon = attacker\GetActiveWeapon!
 
-            if not IsValid(attackerWeapon) return
+            return unless IsValid attackerWeapon
 
-            if not MELEE_WEAPONS[attackerWeapon] return
+            return unless MELEE_WEAPONS[attackerWeapon]
 
-            dmg\ScaleDamage MELEE_DAMAGE_MULT
+            multiplier = getConf "viper_multiplier"
+            dmg\ScaleDamage multiplier
 
     ApplyEffect: =>
         @owner\SetColor PLAYER_COLOR
 
-        timer.Create @timerName, POWERUP_DURATION, 1, -> @Remove!
+        duration = getConf "viper_duration"
+        timer.Create @timerName, duration, 1, @Remove
```

Comment:
This compiles to `timer.Create(self.timerName, duration, 1, self.Remove)`, not passing `self` into `Remove`
```suggestion
        timer.Create @timerName, duration, 1, -> @Remove!
```

Reply:
Ah, knew there was a reason I did it that way